src/components/ADempiere/Form/VPOS2/Options/RMA/index.vue

```diff
@@ -0,0 +1,308 @@
+          label: lang.t('form.pos.tableProduct.quantity'),
+          isNumeric: true,
+          size: '125px'
+        },
+        uom: {
+          columnName: 'UOM',
+          label: lang.t('form.pos.tableProduct.uom'),
+          isNumeric: false,
+          size: '75px'
+        },
+        discount: {
+          columnName: 'Discount',
+          label: lang.t('form.pos.order.discount'),
+          isNumeric: true,
+          size: '100px'
+        },
+        discountTotal: {
+          columnName: 'DiscountTotal',
+          label: lang.t('form.pos.tableProduct.displayDiscountAmount'),
+          isNumeric: true,
+          size: '125px'
+        },
+        discounDisplayTaxIndicator: {
+          columnName: 'taxIndicator',
+          label: lang.t('form.pos.tableProduct.taxRate'),
+          isNumeric: true,
+          size: '80px'
+        },
+        discounDisplayTaxAmounttTotal: {
+          columnName: 'DisplayTaxAmount',
+          label: lang.t('form.pos.tableProduct.taxAmount'),
+          isNumeric: true,
+          size: '150px'
+        },
+        grandTotal: {
+          columnName: 'GrandTotal',
+          label: 'Total',
+          isNumeric: true,
+          isVisible: true,
+          size: '150px'
+        },
+        convertedAmount: {
+          columnName: 'ConvertedAmount',
+          label: lang.t('form.pos.collect.convertedAmount'),
+          isNumeric: true,
+          size: '150px'
+        }
+      }
+    })
+    const searchProduct = ref('')
+    const line = ref({})
+    const lines = computed(() => {
+      return store.getters.getListOrderLines
+    })
+
+    const listLineRMA = computed(() => {
+      return store.getters.getAttributeRMA({
+        attribute: 'listLine'
+      })
+    })
+
+    // Methods
+
+    /**
+     * Query Search
+     */
+    function querySearch(queryString, callback) {
+      var results = queryString ? lines.value.filter(productFilter(queryString)) : lines.value
+      callback(results)
+    }
+
+    function productFilter(queryString) {
+      return (link) => {
+        const search = queryString.toLowerCase()
+        return link.product.value.toLowerCase().includes(search) || link.product.name.toLowerCase().includes(search) || link.product.upc.toLowerCase().includes(search)
+      }
+    }
+
+    function handleSelect(item) {
+      const {
+        id,
+        quantity_ordered
+      } = item
+      store.dispatch('createShipmentLine', {
+        quantity: quantity_ordered.value,
+        orderLineId: id
+      })
+    }
+
+    function selectLine(row, column, event) {
+      const { property } = column
+      if (property === 'quantity') {
+        row.isEditQty = true
+      }
+    }
+
+    function exitLine(row, column, event) {
+      const { property } = column
+      if (property === 'quantity') {
+        row.isEditQty = false
+      }
+    }
+
+    function updateQuantity(quantity) {
+      console.log(quantity, line.value.product.name)
```

Comment:
please remove this console.log

Reply:
https://github.com/solop-develop/frontend-core/pull/1568/commits/f2c16be89c3962b250d28dd3445dd5d5a3ce039c